<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="111664" started-at="2019-08-18T07:47:31Z" finished-at="2019-08-18T07:49:23Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="111664" started-at="2019-08-18T07:47:31Z" finished-at="2019-08-18T07:49:23Z">
      <class name="Facebook.Facebook.FB_Login_page">
        <test-method status="PASS" signature="log_in()[pri:0, instance:Facebook.Facebook.FB_Login_page@754ba872]" name="log_in" duration-ms="19946" started-at="2019-08-18T07:47:31Z" finished-at="2019-08-18T07:47:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- log_in -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Facebook.Facebook.FB_Login_page@754ba872]" name="teardown" is-config="true" duration-ms="2200" started-at="2019-08-18T07:49:20Z" finished-at="2019-08-18T07:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Facebook.Facebook.FB_Login_page -->
      <class name="Facebook.Facebook.Home_Page_Navigation">
        <test-method status="PASS" signature="Home()[pri:0, instance:Facebook.Facebook.Home_Page_Navigation@146ba0ac]" name="Home" duration-ms="27842" started-at="2019-08-18T07:47:54Z" finished-at="2019-08-18T07:48:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Home -->
      </class> <!-- Facebook.Facebook.Home_Page_Navigation -->
      <class name="Facebook.Facebook.Friends_List">
        <test-method status="FAIL" signature="Friends_list()[pri:0, instance:Facebook.Facebook.Friends_List@4dfa3a9d]" name="Friends_list" duration-ms="44051" started-at="2019-08-18T07:48:26Z" finished-at="2019-08-18T07:49:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at Facebook.Facebook.Friends_List.Friends_list(Friends_List.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Friends_list -->
        <test-method status="PASS" signature="test()[pri:0, instance:Facebook.Facebook.Friends_List@4dfa3a9d]" name="test" duration-ms="3468" started-at="2019-08-18T07:49:13Z" finished-at="2019-08-18T07:49:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- Facebook.Facebook.Friends_List -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
